source/sass/component/_slider.scss

```diff
@@ -180,6 +180,31 @@ $c-slider-max-width: calc(#{$container-width-wide} - #{$c-slider-inset}) !defaul
             }
         }
     }
+
+    // Posts as slider
+    &.posts-as-slider {
```

Comment:
Should be formatted as a modifier eg c-slider--post. 

Consider renaming posts to one or more modifiers (multiple modidiers can be usel to define a specific apperance). 

This may be: 

C-slider--arrow-bottom
C-slider--dots 

Modifiers should not remove styling, only append.